### PR TITLE
nad pkg: nad list option added

### PR DESCRIPTION
**Pull request by elenagerman**
nad list option added

### CODE REVIEW

File: pkg/nad/list.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+)
+
+// List returns NADs inventory in the given namespace.
+func List(apiClient *clients.Settings, nsname string) ([]*Builder, error) {
+	if apiClient == nil {
+		glog.V(100).Infof("The apiClient is empty")
+
+		return nil, fmt.Errorf("nadList 'apiClient' cannot be empty")
+	}
+
```
Comment (klaskosk) [inline pkg/nad/list.go:20]: could you please attach the scheme here like in the [Pull function](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/nad/builder.go#L89-L94)

**Review by klaskosk — COMMENTED** [review]
One comment and you could also remove the changes to the schemes package? The schemes should be updated automatically or if necessary in a separate PR

**Comment by elenagerman** [comment]
> One comment and could you also remove the changes to the schemes package? The schemes should be updated automatically or if necessary in a separate PR

thanks Kirsten! I didn't plan to change anything schemes related. I have no idea where this change is came from. Fixed

**Review by klaskosk — APPROVED** [review]
Thanks for fixing the schemes changes, lgtm

**Comment by elenagerman** [comment]
@kononovn @trewest PTAL

**Comment by elenagerman** [comment]
@kononovn @trewest @achuzhoy @cdvultur  PTAL

**Review by cdvultur — APPROVED** [review]
lgtm

**Pull request merged**